addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1933,12 +1933,14 @@ options.registry.SnippetMove = options.Class.extend({
                 if (isNavItem) {
                     $tabPane.prev().before($tabPane);
                 }
+                $('html,body').animate({scrollTop: this.$target.next().position().top - this.$target.innerHeight() - 200}, 500);
```

Comment:
Yes, you can search for the scrollable parent of `this.$target`, it will be more robust.